conf/battle/client.conf

```diff
@@ -169,3 +169,10 @@ macro_detection_punishment: 0
 // Amount of time in minutes that the punishment type is active for. Use 0 for infinite.
 // Official: 0
 macro_detection_punishment_time: 0
+
+// Should the damage timing be synchronized between the client and server?
+// This is not official behavior, but it removes the position lag after being hit.
+// This setting only affects normal monster attacks, unless you set "delay_battle_damage" in battle.conf to "no".
```

Comment:
```
// Do weapon attacks have a attack speed delay before actual damage is applied? (Note 1)
// NOTE: The official setting is yes, even thought it degrades performance a bit.
delay_battle_damage: yes
```
delay_battle_damage is currently not matching the description, as it affects all attacks and not only "weapon attacks".

Reply:
Yeah, it was wrong before already. I didn't really touch the functionality of this feature. Should we update the description as part of this PR?

Reply:
Quick update: I'll rewrite it. Also the new config I move here. So makes more sense when read together.